socialNetwork/socialNetwork/socialNetwork/Controllers/UsersController.cs

```diff
@@ -69,41 +82,44 @@ public async Task<IActionResult> LogIn([FromBody]UserDTO userdata)
         [Route("register")]
         public async Task<IActionResult> Register([FromBody]UserDTO model)
         {
-            var user = new User { UserName = model.Email, Email = model.Email, RememberMe = model.RememberMe, Name = model.Name };
+            var user = new User { UserName = model.Email, Email = model.Email, RememberMe = model.RememberMe, Name = model.Name, ApiKey = model.ApiKey };
```

Comment:
The API key should not be sent from the client, it is created on the backend. The client sees him then and remembers him.